ambry-admin/src/test/java/com.github.ambry.admin/GetReplicasHandlerTest.java

```diff
@@ -62,7 +65,8 @@ public void getReplicasTest() throws Exception {
     List<? extends PartitionId> partitionIds = CLUSTER_MAP.getWritablePartitionIds();
     for (PartitionId partitionId : partitionIds) {
       String originalReplicaStr = partitionId.getReplicaIds().toString().replace(", ", ",");
-      BlobId blobId = new BlobId(partitionId);
+      BlobId blobId = new BlobId(BlobId.DEFAULT_FLAG, ClusterMapUtils.LEGACY_DATACENTER_ID, Account.LEGACY_ACCOUNT_ID,
```

Comment:
I guess since our prod code already starts using this constructor, the tests should be the same. We have a notion that v1 and v2 are purely controlled by `BlobId#CURRENT_VERSION`, so no need to revisit every place (e.g., in this test) when there is a version change.